moduleframework/mtf_scheduler.py

```diff
@@ -188,24 +189,26 @@ def cli():
 class AvocadoStart(object):
     tests = []
     json_tmppath = None
-    additionalAvocadoArg = ''
+    additionalAvocadoArg = []
+    AVOCADO = "avocado"
 
     def __init__(self, args, unknown):
         # choose between TESTS and ADDITIONAL ENVIRONMENT from options
         if args.linter:
-            self.tests.append(
-                "{MTF_TOOLS}/*.py".format(MTF_TOOLS=metadata_common.MetadataLoaderMTF.MTF_LINTER_PATH))
+            self.tests += glob.glob("{MTF_TOOLS}/*.py".format(
+                MTF_TOOLS=metadata_common.MetadataLoaderMTF.MTF_LINTER_PATH))
         self.args = args
 
         for param in unknown:
             # take care of this, see tags for safe/unsafe:
             # http://avocado-framework.readthedocs.io/en/52.0/WritingTests.html#categorizing-tests
-            if os.path.exists(param):
+            testlist = glob.glob(param)
+            if testlist:
                 # this is list of tests in local file
-                self.tests.append(param)
+                self.tests += testlist
```

Comment:
Any reason, why not append?

Reply:
because glob.glob(...) creates list, ``param`` itselfs is string , thats why append has to be replaced by ``+=``